app/models/spree/admin/resources/availability_builder_methods.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Admin
+    module Resources
```

Comment:
Hm, I think `Resources` may be a bit too generic. Wouldn't `Tabs` be more explicit?

Reply:
Changed.